snuba/admin/iam_policy/iam_policy.json

```diff
@@ -39,25 +39,16 @@
           "user:oliver.newland@sentry.io",
           "user:rahul.saini@sentry.io",
           "user:riya.chakraborty@sentry.io",
-          "user:volo.kluev@sentry.io"
+          "user:volo.kluev@sentry.io",
+          "user:david.tsukernik@sentry.io"
```

Comment:
can we have the search and storage group here too rather than listing our individual emails?

Reply:
That part is being worked on. Once we can fetch group membership from the Google API, we can get rid of the individual emails. That's separate from this change however.